### DISCUSSION TITLE
RavenDb community license 12 static indexes limitation

### DISCUSSION THREAD

**Discussion by valeriob** [ravendb/ravendb#17460] [Q&A]
Hi,
I just tested the new release of ravendb 6 and got really surprised to see that the import of a simple database failed, then I saw the limitation of the community license to 12 static indexes.
I think it's quite strong to put so many limitations to the community license, I for instance will not be able to use it in any project anymore, is there any chance that this limitation can be relaxed ?
I think that limiting the "enterprisey" features can be a good trade off, but limiting indexes or sorters it's an auto goal, when you start a project you can't know in advance how many indexes or sorter are going to be needed, so you can't choose ravendb anymore. 
For instance you got simple installation with few data and not fancy things, you can tune the price down using the ravendb community license and grow if/when needed, this is not possible anymore.

Thanks

**Comment by ppekrol** [comment]
Hi @valeriob 

Thanks for feedback. Please contact sales@ravendb.net, I think we can figure sth out together.

**Reply by valeriob** [reply]
Thank you for your feedback, but i'd rather see this solved for everyone rather than per individual, i think is also a better message to everyone who will read this and have the same issues.

**Reply by georgiosd** [reply]
Only speaking for myself but I'm personally very surprised to see a "simple database" have 12 static indexes! Is this something you can open source as a case study?

**Reply by valeriob** [reply]
The scenario is to have an application that you can scale down to match light usage. The application code is the same, but the data is not very much or the users are few or use just one functionality out of many.
